test/cases/api.sh

```diff
@@ -293,16 +316,32 @@ case $(set +x; . /etc/os-release; echo "$ID-$VERSION_ID") in
     ;;
 esac
 
+# Only RHEL need subscription block.
+SUBSCRIPTION_BLOCK=",
+    \"subscription\": {
+      \"organization\": ${API_TEST_SUBSCRIPTION_ORG_ID:-},
+      \"activation-key\": \"${API_TEST_SUBSCRIPTION_ACTIVATION_KEY:-}\",
+      \"base-url\": \"https://cdn.redhat.com/\",
+      \"server-url\": \"subscription.rhsm.redhat.com\",
+      \"insights\": true
+    }"
+
```

Comment:
Can we use a here document and avoid the escape symbols ?

Reply:
Thanks! Good suggestion! Updated.